Firestore/core/src/model/database_id.h

```diff
@@ -46,6 +47,9 @@ class DatabaseId : public util::Comparable<DatabaseId> {
   explicit DatabaseId(std::string project_id,
                       std::string database_id = kDefault);
 
+  /** Returns a DatabaseId from a fully qualified resource name. */
+  static DatabaseId FromName(absl::string_view name);
```

Comment:
Same comment here as other files in `model`.

Reply:
Fixed